mso/resource_mso_schema_site_anp_epg_domain.go

```diff
@@ -65,6 +65,14 @@ func resourceMSOSchemaSiteAnpEpgDomain() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.StringLenBetween(1, 1000),
 			},
+
+			"domain_type_name": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringLenBetween(1, 1000),
+			},
+
 			"dn": &schema.Schema{
```

Comment:
deprecate dn option

Reply:
and add domain_dn and domain_name attributes and add ConflictsWith: []string{"filter_relationship"}, to all of them as discussed.